webpack/vellumUtils.js

```diff
@@ -0,0 +1,83 @@
+/* eslint-env node */
+const fs = require('fs');
+const path = require('path');
+const utils = require('./utils');
+const { emitWarning } = require('node:process');
+
+
+const getAliases = function () {
+    const prodPath = path.resolve(utils.getStaticPathForApp('app_manager', 'js/vellum/'), 'main');
+
+    let aliases = {
+        // Minified version of vellum, used when VELLUM_DEBUG is False
+        "jquery.vellum.prod": prodPath,
+
+        // Supports hqAnalytics in vellum when VELLUM_DEBUG=False
+        "vellum/hqAnalytics": "app_manager/js/forms/form_designer_analytics",
+    };
+
+    // Source version of vellum, used when VELLUM_DEBUG is True
+    const debugDir = getDebugDir();
+    if (debugDir) {
+        aliases['jquery.vellum.dev'] = path.resolve(debugDir, 'src', 'main');
+    } else {
+        // This won't be used by application code, but it needs to resolve to a real path
+        aliases['jquery.vellum.dev'] = prodPath;
+    }
+    return aliases;
+};
+
+const getDebugDir = function () {
+    const dir = '../submodules/formdesigner';
+    try {
+        return fs.realpathSync(path.resolve(__dirname, dir));
+    } catch (e) {
+        if (e.code === "ENOENT") {
+            // This is expected if VELLUM_DEBUG is False
+            emitWarning("Vellum directory not found at " + path);
```

Comment:
Will this warning be printed for anyone running HQ with `VELLUM_DEBUG = False`? If yes, that seems like it could cause confusion. A dev who is not debugging vellum should not get a warning about the vellum debug directory (which they presumably do not care about).

Same question for the similar warning below.

Reply:
I'll just remove it.